### PR TITLE
feature/pd_CreateModels

### PR DESCRIPTION
**Pull request by HAITIANHANK**
Created the relevant User models

### CODE REVIEW

File: Source/BusinessEntities/BusinessEntities/BusinessEntities.csproj [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (CCattano) [inline Source/BusinessEntities/BusinessEntities/BusinessEntities.csproj:1]: Move this csproj under the Adapters solution folder

File: Source/BusinessModels/BusinessModels/BusinessModels.csproj [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (CCattano) [inline Source/BusinessModels/BusinessModels/BusinessModels.csproj:1]: Move this csproj under the WebServices solution folder

File: Source/Data/Blackjack.Data/Entities/UserEntity.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    public class UserEntity
+    {
+        /// <summary>
+        /// The ID number of the user.
+        /// </summary>
+        public int UserID;
+        /// <summary>
+        /// The unique username of the user.
+        /// </summary>
+        public string UserName;
```
Comment (CCattano) [inline Source/Data/Blackjack.Data/Entities/UserEntity.cs:18]: "Username" is one word. Correct casing.

File: Source/BusinessModels/BusinessModels/User/UserBM.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    public class UserBM
+    {
+        /// <summary>
+        /// The ID number of the user.
+        /// </summary>
+        public int UserID;
+        /// <summary>
+        /// The unique username of the user.
+        /// </summary>
+        public string UserName;
```
Comment (CCattano) [inline Source/BusinessModels/BusinessModels/User/UserBM.cs:18]: "Username" is one word. Correct casing.

File: Source/BusinessEntities/BusinessEntities/User/UserBE.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    public class UserBE
+    {
+        /// <summary>
+        /// The ID number of the user.
+        /// </summary>
+        public int UserID;
+        /// <summary>
+        /// The unique username of the user.
+        /// </summary>
+        public string UserName;
```
Comment (CCattano) [inline Source/BusinessEntities/BusinessEntities/User/UserBE.cs:18]: "Username" is one word. Correct casing.

**Review by CCattano — CHANGES_REQUESTED** [review]

File: Source/BusinessEntities/BusinessEntities/BusinessEntities.csproj (hunk 1 above)
Comment (HAITIANHANK) [reply Source/BusinessEntities/BusinessEntities/BusinessEntities.csproj:1]: Should the folder structure Adapters > Business Entities > User 
or something like
Adapters > UserBE

Comment (CCattano) [inline Source/BusinessEntities/BusinessEntities/BusinessEntities.csproj:1]: Rename the csproj file to Blackjack.BusinessEntities for consistency with how we name our other csproj files

File: Source/BusinessModels/BusinessModels/BusinessModels.csproj (hunk 2 above)
Comment (CCattano) [inline Source/BusinessModels/BusinessModels/BusinessModels.csproj:1]: Rename the csproj file to Blackjack.BusinessModels for consistency with how we name our other csproj files

**Review by CCattano — CHANGES_REQUESTED** [review]

**Review by CCattano — APPROVED** [review]

**Pull request merged**